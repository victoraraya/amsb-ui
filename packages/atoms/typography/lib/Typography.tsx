import React from "react";

interface Props {
  children: React.ReactNode;
}

const Typography = ({ children }: Props) => {
  return <p>{children}</p>;
};

export default Typography;
