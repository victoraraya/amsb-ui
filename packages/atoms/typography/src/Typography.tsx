import React from "react";

interface Props {
  children: React.ReactNode;
}
const Typography = ({ children }: Props) => <p>{children}</p>;

export default Typography;
