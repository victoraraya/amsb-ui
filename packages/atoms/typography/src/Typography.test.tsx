import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

import Typography from "./Typography";

it("should render successfully", () => {
  const { asFragment, getByText } = render(<Typography>Hey!</Typography>);

  const typography = getByText("Hey!");

  expect(typography).toBeInTheDocument();

  expect(asFragment()).toMatchSnapshot();
});
