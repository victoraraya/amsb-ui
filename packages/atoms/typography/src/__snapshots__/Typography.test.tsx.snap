// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render successfully 1`] = `
<DocumentFragment>
  <p>
    Hey!
  </p>
</DocumentFragment>
`;
